ui/static/app/record/record-detail.component.ts

```diff
@@ -1,23 +1,27 @@
-import { Component, OnInit } from "@angular/core";
+import { AfterViewInit, Component, ElementRef, OnInit } from "@angular/core";
 import { FormBuilder, FORM_DIRECTIVES, Control, ControlGroup, Validators } from "@angular/common";
 import { CanActivate, Router, RouteParams } from "@angular/router-deprecated";
 import { HTTP_PROVIDERS } from "@angular/http";
+import { AuthService } from "../auth/auth.service";
+import { TooltipDirective } from "../tooltip.directive";
 import { AutocompleteComponent } from "../autocomplete/autocomplete.component";
 import { DomainService } from "../domain/domain.service";
 import { RecordService } from "./record.service";
 import { Record } from "./record";
 import { isLoggedin }  from "../auth/auth.service";
 import "rxjs/add/observable/throw";
 
+declare var jQuery:any;
```

Comment:
Yes, because Bootstrap used jquery, and we used for tooltips. But this line it is to remove.